src/client/components/Pagination/Pagination.component.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useRef, useEffect } from 'react';
+import './Pagination.styles.css';
+
+function Pagination({ num }) {
+  const [floor, setFloor] = useState(num);
+  const level0 = useRef();
+  const level1 = useRef();
+  const level2 = useRef();
+  const level3 = useRef();
+  const level4 = useRef();
+  const level5 = useRef();
+
+  const arr = [level0, level1, level2, level3, level4, level5];
+
+  const handleClick = (number) => {
+    arr.forEach((item) => {
+      if (item.current.style.border) {
+        /* eslint no-param-reassign: "error" */
+        item.current.style.border = 'none';
+      }
+    });
+    setFloor(number);
+  };
```

Comment:
We should not restyle component through manipulating the DOM. please do it the react way it also way simpler :)